### ISSUE TITLE
Bad words

### ISSUE THREAD

**Issue by Jivings** [xyfir/rword#5]
I'm not sure why there is a need for curse words in the list (small.json).

![image](https://user-images.githubusercontent.com/1462828/99792614-05c2a900-2b30-11eb-8ddd-801fb9328d35.png)

Can we remove these or add an option to not return them?

**Comment by MrXyfir** [comment]
I would definitely like to remove these and any others that could potentially be offensive. There are word lists available we could use to strip them out automatically but I don't have the time to do that. Is this something you might be able to do and submit a PR?

- https://github.com/reimertz/curse-words
- https://github.com/MauriceButler/badwords
- https://github.com/LDNOOBW/List-of-Dirty-Naughty-Obscene-and-Otherwise-Bad-Words

**Comment by MrXyfir** [comment]
I removed a bunch of bad/offensive words in v3.2.0

- **big.json**: 370,099 -> 359,742
- **small.json**: 128,660 -> 123,567

I took a very lazy approach to doing this, so surely there are some words I missed and others that were removed that shouldn't have been. Unfortunately most of the lists I've found are very incomplete so I simply combined them all together and then removed any words in big/small that started with one of the bad/offensive words.

If anyone wants to clean this up further I'd be happy to accept a pull request.
